### PR TITLE
add chinese comic source Hanhankuman

### PR DESCRIPTION
**Pull request by feilongfl**

### CODE REVIEW

**Comment by as280093** [comment]
if possible use this icon ba
[ic_launcher).zip](https://github.com/inorichi/tachiyomi-extensions/files/3314508/ic_launcher.3.zip)
sed on source logo and Favicon

**Comment by feilongfl** [comment]
How did you make such a small picture bigger? @as280093

**Comment by as280093** [comment]
just used ms paint and as it was pixel art is was easy

File: src/zh/hanhankuman/build.gradle [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ext {
+    appName = 'Tachiyomi: Hanhankuman'
+    pkgNameSuffix = 'zh.hanhankuman'
+    extClass = '.HanhanKuman'
+    extVersionCode = 1
+    libVersion = '1.2'
+}
+
+dependencies {
+    compileOnly project(':duktape-stub')
```
Comment (arkon) [inline src/zh/hanhankuman/build.gradle]: Remove this if it's not used.

**Pull request merged**